File: backend.Tests/Fixtures.cs
using ResortMap.Api.Models;

namespace ResortMap.Api.Tests;

public static class Fixtures
{
    // Two cabanas: 1-1 and 1-2. A pool tile and a chalet tile are included
    // so MapParser is exercised against every legend character, not just W.
    public const string TinyMap = "....\n.WWc\n.pp.\n....";

    public static List<Guest> Guests => new()
    {
        new Guest("101", "Alice Smith"),
        new Guest("102", "Bob Jones"),
    };
}

File: frontend/src/test/fixtures.ts
import type { MapDto } from '../api';

// Small enough to reason about at a glance: 2 cabanas (one booked, one free),
// one pool tile, one chalet tile, one path tile — exercises every legend
// character without depending on the shape of the real map.ascii.
export const sampleMap: MapDto = {
  width: 4,
  height: 4,
  grid: ['....', '.WWc', '.pp.', '.##.'],
  cabanas: [
    { id: '1-1', row: 1, col: 1, available: true },
    // GET /api/map never returns who booked a cabana (only that it's unavailable) — the
    // real backend keeps room/guestName private, so the fixture mirrors that on purpose.
    { id: '1-2', row: 1, col: 2, available: false },
  ],
};

export const validGuest = { room: '102', guestName: 'Bob Jones' };
